spec/spec.html

```diff
@@ -1,25 +1,30 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Mocha</title>
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="../node_modules/mocha/mocha.css" />
-  </head>
-  <body>
-    <div id="mocha"></div>
-    <script src="../node_modules/mocha/mocha.js"></script>
-    <script src="../node_modules/chai/chai.js"></script>
-    <script src="../node_modules/knockout/build/output/knockout-latest.js"></script>
-    <!-- Application code goes here -->
-    <script src="../lib/calculator.js"></script>
 
-    <script>mocha.setup('bdd');</script>
+<head>
+  <title>Mocha</title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../node_modules/mocha/mocha.css" />
+</head>
 
-    <!-- Test scripts below -->
-    <script src="./calculator.spec.js"></script>
+<body>
+  <div id="mocha"></div>
+  <script src="../node_modules/mocha/mocha.js"></script>
+  <script src="../node_modules/requirejs/require.js"></script>
```

Comment:
We include RequireJS here, but this time without the data-main because we don't want RequireJS to do anything until we tell it to.